lib/src/components/NxColorPicker/NxColorPicker.scss

```diff
@@ -43,14 +39,15 @@
   }
 
   &__color {
-    stroke: none;
     fill: var(--nx-selectable-color-dark);
+    stroke: var(--nx-selectable-color-dark);
+    stroke-width: 1px;
   }
-
+  
```

Comment:
```suggestion
 
```

Reply:
fixed in this commit: [6eab0ba](https://github.com/sonatype/sonatype-react-shared-components/pull/998/commits/6eab0ba8c29b00cb534f8f37a46fe76ffc3422b1)